corehq/apps/data_cleaning/templates/data_cleaning/clean_cases_session.html

```diff
@@ -11,6 +11,9 @@
     <h1 class="fs-3 pe-3 py-3 m-0">
       {{ current_page.page_name }}
     </h1>
+    <div>
+      {% include "data_cleaning/partials/button_bar.html" %}
```

Comment:
Nitpick: Is the div necessary? I'd move it inside the include.

Reply:
yes, it is for the flexbox styling in the parent div. It's easier to understand the layout if it's there, rather than the button bar.